registry.k8s.io/manifests/k8s-staging-node-ipam-controller/promoter-manifest.yaml

```diff
@@ -0,0 +1,10 @@
+# google group for gcr.io/k8s-staging-node-ipam-controller is k8s-infra-staging-node-ipam-controller@kubernetes.io
+registries:
+- name: gcr.io/k8s-staging-node-ipam-controller
+  src: true
+- name: us.gcr.io/k8s-artifacts-prod/node-ipam-controller
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: eu.gcr.io/k8s-artifacts-prod/node-ipam-controller
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: asia.gcr.io/k8s-artifacts-prod/node-ipam-controller
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
```

Comment:
This is incorrect. We now use Artifact Registry repos:
https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/manifests/k8s-staging-apisnoop/promoter-manifest.yaml